cfme/fixtures/vm.py

```diff
@@ -9,13 +13,68 @@
 from cfme.utils.wait import wait_for
 
 
+DOCSTRING_CACHE = {}
+TEST_PARAMS = re.compile(r"\[.*\]")
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        '--no-assignee-vm-name',
```

Comment:
maybe something shorty here like `--no-vm-name`? See what other people think.

Reply:
I think we are keeping this as is for now unless @mshriver wants to change it to `--no-vm-auto-naming` ??

Reply:
We'll just keep it and collectively hate it forever.